OnlineTicketBooking/src/main/webapp/WEB-INF/images/dbschema.txt

```diff
@@ -0,0 +1,70 @@
+

```

Comment:
create a folder with name dbscripts and place the file in it.

Reply:
ok sir